test/resources/sampleContainer/example1.ttl

```diff
@@ -2,6 +2,8 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/> .
 @prefix ex: <http://example.org/stuff/1.0/> .
 
+</sampleContainer/example*> a :glob.
```

Comment:
The example tests the serialization base being the glob request uri. The one you mention, `<#a> dc:title "foo" .` tests the parsing base (relative to example1.ttl). I'll add this line also.